docs/platforms/native/advanced-usage/backend-tradeoffs/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Backend Tradeoffs
+description: "How to choose the right crash backend in the Native SDK."
+sidebar_order: 1000
+---
+The Native SDK lets users decide at compile-time between three crash backends:
+
+* `crashpad`
+* `breakpad`
+* `inproc`
+
+Currently, `crashpad` is the default on all desktop platforms because it
+
+* has an external `handler` process that allows for external snapshots and sending crash reports immediately (instead of on the next successful start of your application)
+* is the primary target for extension compared to upstream, including
```

Comment:
It's late so maybe that's why but I don't understand this sentence

specifically "target for extension compared to upstream", what's upstream in this case? and what extension are you referring to?

Reply:
In this case, "upstream" is the Google `crashpad` repository, and "extensions" are changes _we_ introduced to make `crashpad` more useful for customers.

What I wanted to express here is that most changes, compared to what the upstream repository already provides, flow into `crashpad`. This includes breaking compatibility with the upstream repository (you can no longer build or run the Native SDK with the upstream `crashpad[_handler]`).

This is not necessarily good or bad because some changes - as you mentioned below - are necessary due to `crashpad`'s shortcomings concerning our (customers') use cases. 

However, we are investing more time adapting the backend to our needs than the other two. Some "extensions" aren't currently available in the other two, even if they could make sense there.

I missed adding this context, which would make the text a bit harder to read. The wording with "upstream" and "extensions" is also unclear. I am open to suggestions on how to reformulate this.

We can remove the "extensions" point entirely, although it visualizes both improvements and our investment in the backend. We must also be careful about separating the current state of affairs regarding backends (which I attempted to document here) from contextual questions like "What do we want to achieve with the Native SDK, and where do we want to go?".